src/organisms/cards/RadioCard/index.js

```diff
@@ -132,7 +165,15 @@ RadioCard.propTypes = {
   /**
    * bold label displayed in card
    */
-  label: PropTypes.string.isRequired,
+  label: (props) => {
+    if (!props['label'] && !props['renderProp']) {
+      return new Error(
+        'Label prop is required',
+      );
+    }
+
+    return null;
+  },
```

Comment:
Woah. This is pretty cool - a custom propType?

Reply:
yeah, added it because `label` should still be required but only when `renderProp` is not defined in the props